docs: panel refactor initial build out (#41840)
grafana/grafana: docs/sources/panels-refactor/add-panels-dynamically/_index.md

+++

> title = "Enable template variables to add panels dynamically"

Reviewer: ```suggestion
title = "Add panels dynamically"
```

> aliases = ["/docs/grafana/latest/features/panels/panels/"]

Reviewer: Is the duplicated `panels` subdirectory correct?

weight = 200
+++

> # Enable template variables to add panels dynamically

Reviewer: ```suggestion
# Add panels dynamically
```

> Harness the power of Grafana by configuring the system to automatically and dynamically add panels based on criteria that you define.

Reviewer: ```suggestion
Harness the power of Grafana by enabling template variables to automatically and dynamically add panels based on criteria that you define.
```
